package.json

```diff
@@ -31,7 +31,7 @@
     "dnssd": "^0.4.1",
     "express": "^4.16.3",
     "express-ws": "^4.0.0",
-    "local-ipv4-address": "^0.0.2"
+    "internal-ip": "^3.0.1"
```

Comment:
This should have a corresponding change in package-lock.json which is at least part of why Travis is sad about this PR

Reply:
Fixed.